scripts/rosdistro_freeze_source

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+from rosdistro.index import Index
+
+import argparse
+import os.path
+import subprocess
+import sys
+import threading
+import time
+import yaml
+
+try:
+    import queue
+except ImportError:
+    import Queue as queue
+
+from rosdistro import get_distribution
+from rosdistro.writer import yaml_from_distribution_file
+
+CONCURRENT_OPERATIONS=16
```

Comment:
I think it would be good to make it a command line argument. Even if nobody will ever change it it nicely makes that detail visible to the user. But you call - since I might not use the script anyway :wink: